soci/ztoc_builder.go

```diff
@@ -69,7 +71,7 @@ func BuildZtoc(gzipFile string, span int64, cfg *buildConfig) (*Ztoc, error) {
 	}
 
 	return &Ztoc{
-		Version:              "0.1",
+		Version:              "0.9",
```

Comment:
0.9?

Reply:
Honestly, I was reluctant to put 1.0 just yet. The ztoc doesn't change, e.g. we do not add/change any fields. The only change is how it is serialized. We can put 1.0 upon concluding on the ztoc format.